packages/react-router-dev/vite/node-adapter.ts

```diff
@@ -86,7 +86,11 @@ export function fromNodeRequest(
 // https://github.com/solidjs/solid-start/blob/7398163869b489cce503c167e284891cf51a6613/packages/start/node/fetch.js#L162-L185
 export async function toNodeRequest(res: Response, nodeRes: ServerResponse) {
   nodeRes.statusCode = res.status;
-  nodeRes.statusMessage = res.statusText;
+
+  // HTTP/2 doesn't support status messages (RFC7540 8.1.2.4)
```

Comment:
Lets drop a link in here for easier lookup

```suggestion
  // HTTP/2 doesn't support status messages
  // https://datatracker.ietf.org/doc/html/rfc7540#section-8.1.2.4
```